tests/e2e/Services/Locale/LocaleBase.php

```diff
@@ -159,8 +159,8 @@ public function testGetContinents():array
         $this->assertEquals($response['headers']['status-code'], 200);
         $this->assertEquals(7, $response['body']['sum']);
         $this->assertIsArray($response['body']['continents']);
-        $this->assertEquals($response['body']['continents'][0]['code'], 'NA');
-        $this->assertEquals($response['body']['continents'][0]['name'], 'América del Norte');
+        $this->assertEquals($response['body']['continents'][4]['code'], 'NA');
```

Comment:
We should make sure we're sorting the list using the locale ABC

Reply:
Done :+1: